src/main/java/gregtech/loaders/MaterialInfoLoader.java

```diff
@@ -14,415 +15,442 @@
-                new MaterialStack(Materials.Rubber, M * 2))); // plate
-
-        OreDictUnifier.registerOre(MetaTileEntities.HULL[3].getStackForm(), new ItemMaterialInfo(
-                new MaterialStack(Materials.StainlessSteel, M * 8), // plate
-                new MaterialStack(Materials.Gold, M), // single cable
-                new MaterialStack(Materials.Rubber, M * 2))); // plate
-
-        OreDictUnifier.registerOre(MetaTileEntities.HULL[4].getStackForm(), new ItemMaterialInfo(
-                new MaterialStack(Materials.Titanium, M * 8), // plate
-                new MaterialStack(Materials.Aluminium, M), // single cable
-                new MaterialStack(Materials.Rubber, M * 2))); // plate
-
-        OreDictUnifier.registerOre(MetaTileEntities.HULL[5].getStackForm(), new ItemMaterialInfo(
-                new MaterialStack(Materials.TungstenSteel, M * 8), // plate
-                new MaterialStack(Materials.Platinum, M), // single cable
-                new MaterialStack(Materials.Rubber, M * 2))); // plate
-
-        OreDictUnifier.registerOre(MetaTileEntities.HULL[6].getStackForm(), new ItemMaterialInfo(
-                new MaterialStack(Materials.RhodiumPlatedPalladium, M * 8), // plate
-                new MaterialStack(Materials.NiobiumTitanium, M), // single cable
-                new MaterialStack(Materials.Rubber, M * 2))); // plate
-
-        OreDictUnifier.registerOre(MetaTileEntities.HULL[7].getStackForm(), new ItemMaterialInfo(
-                new MaterialStack(Materials.NaquadahAlloy, M * 8), // plate
-                new MaterialStack(Materials.VanadiumGallium, M), // single cable
-                new MaterialStack(Materials.Rubber, M * 2))); // plate
-
-        OreDictUnifier.registerOre(MetaTileEntities.HULL[8].getStackForm(), new ItemMaterialInfo(
-                new MaterialStack(Materials.Darmstadtium, M * 8), // plate
-                new MaterialStack(Materials.YttriumBariumCuprate, M), // single cable
-                new MaterialStack(Materials.Rubber, M * 2))); // plate
-
-        OreDictUnifier.registerOre(MetaTileEntities.HULL[9].getStackForm(), new ItemMaterialInfo(
-                new MaterialStack(Materials.Neutronium, M * 8), // plate
-                new MaterialStack(Materials.Europium, M), // single cable
-                new MaterialStack(Materials.Rubber, M * 2))); // plate
-
-
-        OreDictUnifier.registerOre(MetaTileEntities.ENERGY_INPUT_HATCH[3].getStackForm(), new ItemMaterialInfo(
-                new MaterialStack(Materials.StainlessSteel, M * 8), // plate
-                new MaterialStack(Materials.Gold, M * 2), // single cable
-                new MaterialStack(Materials.Rubber, M * 4), // plate
-                new MaterialStack(Materials.BlackSteel, M * 2), // fine wire
-                new MaterialStack(Materials.SteelMagnetic, M / 2) // rod
+        OreDictUnifier.registerOre(MetaTileEntities.HULL[GTValues.ULV].getStackForm(), new ItemMaterialInfo(
```

Comment:
If you use Assembler to craft Mahine Hulls, these recycle recipes will cause plate dupe
(you don't need wrought iron plates to craft machine hulls in assembler!)

Reply:
This statement is correct but it's example (part in brackets) is applicable only to LV hull. Though the problem is present on probably all of them.